shared/types/user.ts

```diff
@@ -21,6 +21,7 @@ export const UserBase = z.object({
       payment: z.boolean().optional(),
       children: z.boolean().optional(),
       postmanSms: z.boolean().optional(),
+      fasterDownloads: z.boolean().optional(),
```

Comment:
Shouldn't need this change since we're using growthbook (FE flag)

Reply:
Fixed!